demos/mqtt/mqtt_demo_basic_tls/Makefile

```diff
@@ -1,7 +1,8 @@
 CC = gcc
 
 INCLUDE_DIRS = -I . \
-               -I ../../../libraries/standard/mqtt/include
+               -I ../../../libraries/standard/mqtt/include \
+               -I ../../logging-stack/
```

Comment:
do we have to maintain the Makefiles as we have migrated to cmake?

Reply:
Possibly not, but I don't think we're ready to remove them yet. If anyone's using a Mac then CMake won't work but the Makefile should